docs/TuxOwnerClass.md

```diff
@@ -0,0 +1,307 @@
+As with any other `React` class, the `ownerClassProps` can optionally include `React` [lifecycle methods and specs](#http://facebook.github.io/react/docs/component-specs.html).
+
+```
+    ownerClassProps.componentDidMount = function () {
+      roomActions.get();
+    };
+```
+
+***
+
+##### <a id="ownerClassProps-connectOwnerToStore"></a>3.1) Method - `ownerClassProps.connectOwnerToStore` - type: FUNCTION or ARRAY of FUNCTIONs - optional [#](#ownerClassProps-connectOwnerToStore)
+This method allows an `OwnerClass` to easily and semantically add and remove event listeners for stores. The listeners will be added on `componentDidMount` and will be removed on `componentWillUnmount`. The `connectOwnerToStore` method must return an object which the `OwnerClass` will use to connect itself to the store. Alternatively, `connectOwnerToStore` can be an array of functions that return objects if the `OwnerClass` needs to connect to multiple stores. In either case, the function(s) will be invoked with the context of the `Owner Component` just as with any other `React` spec or lifecycle method.
+
+```
+    ownerClassProps.connectOwnerToStore = function () {
+      //'this' will be the Owner Component so we can invoke methods like this.setState
+      return connectOwnerToStoreProps;
+    };
+
+    connectOwnerToStoreProps = {
+      store: roomStore,
+      listener: [ownerClassProps.onRoomStoreChange, ownerClassProps.onRoomStoreSpecialChange],
+      event: ['CHANGE', 'SPECIAL_CHANGE']
+    };
+```
+
+Alternatively, to connect an `OwnerClass` to multiple stores:
+
+```
+    ownerClassProps.connectOwnerToStore = [
+      function () {
+        return connectOwnerToStoreProps;
+      },
+
+      function () {
+        return connectOwnerToAnotherStoreProps;
+      }
+    ];
+```
+
+The `connectOwnerToStoreProps` object is described in full below.
+
+###### <a id="connectOwnerToStoreProps-store"></a>Property - `connectOwnerToStoreProps.store` - type: OBJECT - required [#](#connectOwnerToStoreProps-store)
+The store to add our listeners to.
+
+###### <a id="connectOwnerToStoreProps-listener"></a>Property - `connectOwnerToStoreProps.listener` - type: FUNCTION or ARRAY of FUNCTIONs - required [#](#connectOwnerToStoreProps-listener)
+Listener callback (or array of listener callbacks) to invoke when the corresponding events are emitted from the store. If there is no `event` key then the listeners will be triggered when the default `CHANGE` event is emitted. If there is an `event` key than the listener will be mapped to that event. If an array of listeners is passed in than each listener will be mapped to the corresponding `event` in the event array. If the listener array is longer than the event array or only a single event is passed in than each extra listener will be mapped to the single event or last event in the array.
```

Comment:
For the mentions of event in this line, would it make sense to say 'event type' where applicable?